stable/oauth2-proxy/values.yaml

```diff
@@ -5,16 +5,28 @@ config:
   # OAuth client secret
   clientSecret: "XXXXXXXX"
   # Create a new secret with the following command
-  # python -c 'import os,base64; print base64.b64encode(os.urandom(16))'
+  # openssl rand -base64 32 | head -c 32 | base64
+  # Use an existing secret for OAuth2 credentials (see secret.yaml for required fields)
+  # Example:
+  # existingSecret: secret
   cookieSecret: "XXXXXXXXXX"
+  google: {}
+    # adminEmail: xxxx
+    # serviceAccountJson: xxxx
+    # Alternatively, use an existing secret (see google-secret.yaml for required fields)
+    # Example:
+    # existingSecret: google-secret
+  # Default configuration, to be overridden
+  configFile: |-
+    email_domains = [ "*" ]
+    upstreams = [ "file:///dev/null" ]
   # Custom configuration file: oauth2_proxy.cfg
   # configFile: |-
   #   pass_basic_auth = false
   #   pass_access_token = true
-  configFile: ""
-  # Use an existing secret for OAuth2 credentials
+  # Use an existing config map (see configmap.yaml for required fields)
   # Example:
-  # existingSecret: secret
```

Comment:
`config.existingSecret` is still a valid thing no?

Reply:
It was moved a few line above actually, the diff doesn't show it well I believe but it still there :)